app/scripts/actions/index.js

```diff
@@ -332,18 +333,36 @@ export const getApiVersion = () => {
 export const checkApiVersion = () => {
   return (dispatch, getState) => {
     const { versionNumber } = getState().apiVersion;
-    if (compatibleApiVersion.indexOf(versionNumber) < 0) {
+    const apiVersionParsed = parseVersionString(versionNumber);
+    let isCompatible = false;
+    for (let version of minCompatibleApiVersions) {
+      const parsedVersion = parseVersionString(version);
+      if (apiVersionParsed.major === parsedVersion.major &&
+          apiVersionParsed.minor === parsedVersion.minor &&
+          apiVersionParsed.patch >= parsedVersion.patch) {
```

Comment:
It seems like this kind of comparison would fail if the API version was `1.0.0-alpha`. Maybe we should just use a package to handle this sort of thing? I just found https://www.npmjs.com/package/compare-versions